docs/interactions/bots-api.md

```diff
@@ -0,0 +1,92 @@
+# Bots API
+
+Engage Digital’s Bots API is our way to integrate bots into Engage Digital chat channels. This guide is meant to provide integrators a comprehensive overview of the available features for them to be able to create an implementation.
+
+Currently it's only available on Messenger channels, their documentation is available [here](https://developers.facebook.com/docs/messenger-platform/handover-protocol).
+
+## Setup
+
+Please contact Support to get access to the feature on ED side.
```

Comment:
```suggestion
```

If we publish the doc, we will remove the feature from beta

Reply:
Ok do we have any info on when that will be ?

Reply:
will be handled internally